pkg/controller/virtinformers.go

```diff
@@ -167,6 +172,15 @@ func (f *kubeInformerFactory) OfflineVirtualMachine() cache.SharedIndexInformer
 	})
 }
 
+func (f *kubeInformerFactory) ConfigMap() cache.SharedIndexInformer {
+	// We currently only monitor configmaps in the kube-system namespace
+	return f.getInformer("configMapInformer", func() cache.SharedIndexInformer {
+		restClient := f.clientSet.CoreV1().RESTClient()
+		lw := cache.NewListWatchFromClient(restClient, "configmaps", systemNamespace, fields.Everything())
+		return cache.NewSharedIndexInformer(lw, &k8sv1.ConfigMap{}, f.defaultResync, cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
```

Comment:
Could you try to use a fieldselector, like described here: https://github.com/kubernetes/apimachinery/blob/b317fa7ec8e0e7d1f77ac63bf8c3ec7b29a2a215/pkg/apis/meta/v1/helpers.go#L204? May or may not work. If yes I would prefer it.

Reply:
Definitely! I just saw how @davidvossel did it in `tests/ovm_test.go`, I'll crib off his work.

Reply:
you can watch for only one map. checked it. That is preferrable. See above.